app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# require 'pry'
+class SessionsController < ApplicationController
+
+  def new
+    @user
+  end
```

Comment:
I don't understand what there is a variable here that isn't set to anything. Are you using this somehow? If so, set it to something. If not, you can get rid of it.